src/components/Header/Header.tsx

```diff
@@ -1,20 +1,27 @@
 import React from 'react'
 import { NavLink, Link } from 'react-router-dom'
 import { slide as Menu } from 'react-burger-menu'
-import './Header.scss'
 
 import Search from '../Search/Search'
+import { OpenMenuContext } from '../../contexts/index'
+import './Header.scss'
 
 const Header: React.FC = () => {
 	return (
 		<header className="Header">
-			<Menu
-				customBurgerIcon={<img src="../search.svg" alt="search icon" />}
-				customCrossIcon={<img src="../close.svg" alt="close icon" />}
-				width="min-content"
-			>
-				<Search />
-			</Menu>
+			<OpenMenuContext.Consumer>
+				{({ isOpen, stateChangeHandler }) => (
```

Comment:
@amykarnaze we found that you can destructure here instead of using `useContext`

Reply:
I believe it's destructuring implicit props form the provider.